src/runtime/image.ts

```diff
@@ -95,15 +96,16 @@ function resolveImage (ctx: ImageCTX, input: string, options: ImageOptions): Res
   const { provider, defaults } = getProvider(ctx, options.provider || ctx.options.provider)
   const preset = getPreset(ctx, options.preset)
 
-  const _options = { ...defaults, ...preset, ...options }
+  const _options: ImageOptions = defu({ modifiers: {} }, options, preset, defaults)
+
   if (_options.modifiers?.width) {
     _options.modifiers.width = parseSize(_options.modifiers.width)
   }
   if (_options.modifiers?.height) {
     _options.modifiers.height = parseSize(_options.modifiers.height)
   }
 
-  const image = provider.getImage(input, JSON.parse(JSON.stringify(_options)), ctx)
+  const image = provider.getImage(input, _options, ctx)
```

Comment:
You still need ` const image = provider.getImage(input, JSON.parse(JSON.stringify(_options)), ctx)` otherwise it doesn't work. Please run locally `rm -rf ./playground/node_modules/ && rm -rd ./playground/dist/ && npx nuxt generate playground` and you will see.

Reply:
I see, I guess changing `modifiers` reference should be enough.

Reply:
@farnabaz it works :).